### PR TITLE
- Don't set series and episode value if neither value is greater  than zero

### PR DESCRIPTION
**Pull request by rbarthen**
I discovered that ArgusTV will default series number to 0 when not available. As a result, when series 0 is passed and no episode is passed, we get some strange title formatting where it will display "S0" and sometimes not have any white space after it.  Therefore, series and episode should only be populated if series or episode has a value greater than zero (which indicates it isn't just using a default number).

### CODE REVIEW

File: src/pvrclient-argustv.cpp
Comment (ksooo) [inline src/pvrclient-argustv.cpp]: Coding guidelines: blank after ‚if‘ and blank after ,>‘, please

**Comment by ksooo** [comment]
May I suggest that you first only open a PR for Nexus and then, once you have approval for this PR, you open the respective Matrix PR. Saves us all some time. You have no double work with applying requested changes to two PRs and we do not get so many notifications from GitHub and most important we have exactly one place to review and discuss the matter.

**Comment by rbarthen** [comment]
> Max I suggest that you first only open a PR for Nexus and then, once you have approval for this PR, you open the respective Matrix PR. Saves us all some time. You have no double work with applying requested changes to two PRs and we do not get so many notifications from GitHub and most important we have exactly one place to review and discuss the matter.

This makes sense. I will follow this process if I create any other PRs.

File: src/pvrclient-argustv.cpp
Comment (rbarthen) [reply src/pvrclient-argustv.cpp]: > Coding guidelines: blank after ‚if‘ and blank after ,>‘, please

Sorry, I had cleaned that up in the Matrix PR, but failed to clean it up here.  It is cleaned up now.

**Review by phunkyfish — APPROVED** [review]

**Pull request merged**
